### PR TITLE
A couple of fixes

### PR DESCRIPTION
**Pull request by smurfix**
See individual changes.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 
-from pathlib import Path
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
```
Comment (WhyNotHugo) [inline setup.py]: This would only affect Python < 3.4, but we really just support Python >= 3.5.

Does this actually make Python 3.4 work?

**Comment by smurfix** [comment]
On 14.05.2018 17:07, Hugo Osvaldo Barrera wrote:
>
> This would only affect Python < 3.4, but we really just support Python
> >= 3.5.
>
While I don't have 3.4 on my machine any more, based on the fact that it
works with 2.7 the answer should be Yes.

$ PYTHONPATH=. python build/python-barcode create -t png --text foo hello /tmp/test
New barcode saved as /tmp/test.png.
$ python --version
Python 2.7.13
$ eog /tmp/test.png
[ observe perfectly reasonable barcode ]
$ 

-- 
-- Matthias Urlichs

**Comment by WhyNotHugo** [comment]
TBH, I don't really care about adding support for python2 at this point in time.

I [and the community in general] am moving towards dropping python2 support, so, even thought it's "just three extra lines", it's just something that I don't see making much sense.

I'd be happy to merge the rest of this PR (which looks great), but I'm not really in favour of python2 right now.

**Comment by smurfix** [comment]
Fair enough. Feel free to cherry-pick the other two then ;-)

**Comment by WhyNotHugo** [comment]
Closing; those two commits have been cherry-picked into master.

**Comment by WhyNotHugo** [comment]
Thanks!

**Pull request closed without merging**